generics.R

```diff
@@ -1,20 +1,4 @@
 # Generics to support LightGBM model type
-vetiver_create_description._lgb.Booster <- function(model) {
-  run_id <- Sys.getenv("AWS_S3_MODEL_RUN_ID", unset = "Unknown")
-  paste("Run ID:", run_id)
-}
-
-
-vetiver_create_meta._lgb.Booster <- function(model, metadata) {
-  vetiver_meta(metadata, required_pkgs = c("lightgbm", "lightsnip"))
-}
-
-
-handler_startup._lgb.Booster <- function(vetiver_model) {
-  attach_pkgs(vetiver_model$metadata$required_pkgs)
-}
```

Comment:
I believe these helper functions are no longer necessary since we're not calling the underlying functions in the same way, but I could be wrong!